_posts/newsletters/2021-03-22-newsletter.md

```diff
@@ -0,0 +1,126 @@
+
+<a name="communitycall"></a>
+# **US-RSE Community Calls**
+
+The topic of the March community call was "**Changing How Academia views RSEs**." We discussed questions like how different institutions see the role of RSEs in research, how the work of RSEs is being recognized, and career progression and retention. We created a template for a white paper; stay tuned for follow-up activities! The next community call will take place April 8th. You can suggest and vote on topics [on GitHub](https://github.com/USRSE/monthly-community-calls/issues).
+
+<a name="council"></a>
+# **International Council of RSE Associations**
+
+US-RSE is one of the founding members of the [International Council of RSE Associations](https://researchsoftware.org/2021/01/27/introducing-the-international-council-of-RSE-associations.html). Formed during the 2nd International RSE leaders Meeting, the Council provides a forum for leaders from established national and multinational RSE associations to communicate in order to better collaborate on international events, share knowledge and experiences, and resolve issues. The council met for the first time in Jan 2021 and has its third meeting scheduled for next week. Topics currently being discussed involve a coordinated international RSE event for 2021, an international web presence, and the overall Council charter. The Council will be the [topic of an upcoming talk](https://sorse.github.io/programme/finale/) at the SORSE finale.
+
+<a name="panel"></a>
+# **ADSA/US-RSE Career Panel**
+
+Last month, in partnership with [The Academic Data Science Alliance](https://academicdatascience.org/), we hosted an early career panel: "What do data scientists and research software engineers do in academia?" We heard from data scientists and research software engineers about their career paths, where and how to find these positions, and what to expect when working in these fields. For those who were unable to attend, a [recording of the event](https://www.youtube.com/watch?v=AW6nIptCVAw) is available on the US-RSE [YouTube channel](https://www.youtube.com/channel/UC7IQsWv809OQYJ-sJKuQZrw).
+
+<a name="geohelp"></a>
+# **Geosciences Data & Software Help Desk Opportunity**
+
+Geoscientists want to improve their data and software practices, but sometimes aren't sure how.  You can help. **Please consider volunteering for the upcoming [Virtual](http://bit.ly/DataHelpEGU21) [Data Help Desk](http://bit.ly/DataHelpEGU21)** to be held during the 2021 European Geosciences Union General Assembly (vEGU21). You do not have to be attending vEGU21 to take part. Resources created for this event can of course be reused for future Data Help Desks - contact megancarter@esipfed.org if you'd like to discuss this opportunity.
```

Comment:
If we linked to emails previously (lance's) I think it would be consistent to do it here as well.

Reply:
Done.